src/jrd/RecordSourceNodes.h

```diff
@@ -1081,6 +1081,24 @@ class UnlistFunctionSourceNode : public TableValueFunctionSourceNode
 	}
 };
 
+class GenSeriesFunctionSourceNode : public TableValueFunctionSourceNode
```

Comment:
I'd say it's better to declare the class as `final` instead of each method. Methods should be declared with `override`.